src/panels/config/devices/ha-config-device-page.ts

```diff
@@ -185,6 +185,27 @@ export class HaConfigDevicePage extends LitElement {
         )
   );
 
+  private _getEntitiesSorted = (entities) =>
```

Comment:
```suggestion
  private _getEntitiesSorted = (entities : HassEntity[]) =>
```

Reply:
HassEntity should be imported, right?
Will check and test it when be near a PC, these days have no this luxury(